WooCommerce/Classes/ViewModels/Order Details/CardPaymentReadinessUseCase.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import Combine
+import Yosemite
+
+final class CardPaymentReadinessUseCase {
```

Comment:
I was going to suggest naming this CardPresentPaymentsReadinessUseCase, but it's too long. 

On the one hand, we have moved to word the feature In-Person Payments publicly, on the other, all our code still calls it CardPresentPayments.

So, maybe IPPReadinessUseCase? I don't know. Definitely not a blocker.

Reply:
Yeah can do. I was trying to avoid using CPP and IPP abbreviations, but your right to call this out as "Card Payment" isn't specific enough for what this checks. 

On balance, even though it's long, I prefer `CardPresentPaymentsReadinessUseCase` for the specificity and consistency with the rest of our code.